stats/handlers.go

```diff
@@ -45,13 +45,15 @@ type ConnTagInfo struct {
 	RemoteAddr net.Addr
 	// LocalAddr is the local address of the corresponding connection.
 	LocalAddr net.Addr
-	// TODO add QOS related fields.
 }
 
 // RPCTagInfo defines the relevant information needed by RPC context tagger.
+// FailFast is only valid on client side, it's always false on server side.
```

Comment:
I think it would be more appropriate to document this on the field than on the struct.

How far do we think we will be extending this struct?  Would it make sense to have sub-structures for client- and server- only fields?